d3pie/d3pie.d.ts

```diff
@@ -0,0 +1,144 @@
+// Type definitions for d3pie 0.1.9
+// Project: https://github.com/benkeen/d3pie
+// Definitions by: Petryshyn Sergii <https://github.com/mc-petry>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface ID3PieChart {
```

Comment:
http://definitelytyped.org/guides/best-practices.html#ghost-modules
Could you use this pattern?